### ISSUE TITLE
Add support to workspace dictionary

### ISSUE THREAD

**Issue by jiaqi61** [valentjn/vscode-ltex#95]
**Is your feature request related to a problem? Please describe.**
I often work on several latex projects and there are some words that I hope them be counted as spelling errors in one project while not be regarded as errors in another project.

**Describe the solution you'd like**
Add a workspace dictionary like the one in code spell checker, where we can add words to the whitelist only for the current workspace.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by valentjn** [comment]
Thanks for the feature request! This is definitely a desirable feature for LT<sub>E</sub>X. Theoretically, this is already possible. However, I just tested it, and it doesn't seem to work the way I imagined it (I somehow thought the dictionaries from user and workspace settings would be merged magically). This makes some architectural changes in the configuration necessary. So it's not straightforward, but it will be implemented.
